src/main/kotlin/kotlinx/kover/engines/jacoco/JacocoAgent.kt

```diff
@@ -29,17 +28,24 @@ internal class JacocoAgent(val config: Configuration, private val project: Proje
         val binary = extension.binaryReportFile.get()
         binary.parentFile.mkdirs()
 
-        return listOf(
+        return listOfNotNull(
             "destfile=${binary.canonicalPath}",
             "append=false", // Kover don't support parallel execution of one task
             "inclnolocationclasses=false",
             "dumponexit=true",
             "output=file",
-            "jmx=false"
+            "jmx=false",
+            extension.includes.filterString("includes"),
+            extension.excludes.filterString("excludes")
         ).joinToString(",")
     }
 }
 
+private fun List<String>.filterString(name: String): String? {
+    if (isEmpty()) return null
```

Comment:
`isBlank` or `isEmpty`?

Reply:
In this case, I'm working with a `List`, not a `String`. 
If no filters are defined (an empty list), we return null so that it is not passed to the agent arguments.